mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/components/SlippageToleranceV2.tsx

```diff
@@ -96,6 +91,23 @@ export function SlippageToleranceV2({
     );
   }, [selectedSlippage]);
 
+  const updateSlippage = (slippage: string, updateLocalStorage: boolean) => {
+    // update local storage
+    if (updateLocalStorage) {
```

Comment:
I think this feature have changed. Is the expected behavior should be:

- Whatever the user used, this will be the "default" slippage. Meaning, it will be saved on local storage everytime. Regardless of custom or non-custom.

Reply:
The if condition is to make sure that the local storage only updates when the set button is pressed for custom
<img width="394" alt="image" src="https://user-images.githubusercontent.com/13292662/190947327-c86fbf99-31d2-4c3b-a67b-a63adfa90edc.png">